matrix-sdk-android/src/main/java/im/vector/matrix/android/api/session/events/model/Event.kt

```diff
@@ -81,6 +84,7 @@ data class Event(
 
     var mxDecryptionResult: OlmDecryptionResult? = null
     var mCryptoError: MXCryptoError.ErrorType? = null
+    var sendState: SendState = SendState.UNKNOWN
```

Comment:
Why did you pass it in Event class? We should avoid adding more "var" not in spec...